### ISSUE TITLE
Exporting accounts using secure URL format fails with TypeError (crypto module)

### ISSUE THREAD

**Issue by lostpebble** [near/wallet-selector#680]
I've started testing out the account export implementation with Meteor today and ran into this issue:

```
Uncaught TypeError: crypto.randomBytes is not a function
    at getNextRandomValue (generate.js:11:24)
    at randomNumber (generate.js:23:13)
    at generate (generate.js:49:20)
    at module.exports.generate (generate.js:138:17)
    at generateSecretKey (index.js:4946:43)

Uncaught TypeError: Cannot read properties of undefined (reading 'length')
    at getNextRandomValue (generate.js:9:62)
    at randomNumber (generate.js:23:13)
    at generate (generate.js:49:20)
    at module.exports.generate (generate.js:138:17)
    at generateSecretKey (index.js:4946:43)
    at index.js:5005:17

The above error occurred in the <Passphrase> component:

    at Passphrase (http://localhost:4200/node_modules/.vite/deps/@near-wallet-selector_account-export.js?v=d2207208:4506:3)
    at ExportAccount (http://localhost:4200/node_modules/.vite/deps/@near-wallet-selector_account-export.js?v=d2207208:4713:3)
```

It appears to be an issue with the `generate-password` library (https://github.com/brendanashworth/generate-password) which is used to generate the secret key to encrypt the user's accounts and private keys for exports:

```ts
export const generateSecretKey = (): string =>
  generator.generate({
    length: nacl.secretbox.keyLength,
    numbers: true,
    strict: true,
    lowercase: true,
    uppercase: true,
    symbols: true,
  });
```

On their docs, they say that it is not browser compatible (since they `require()` a Node.js module), which means it can't serve our purposes here. They do mention a browser-compatible version of their library though: https://www.npmjs.com/package/generate-password-browser.

But- I'm not really sure why we are adding this as a dependency though? The Near API JS library already has the ability to generate random keys on the fly, right? :

```ts
import { KeyPair } from "near-api-js";

const randomKey = KeyPair.fromRandom("ed25519").getPublicKey().toString();
```

Could that not serve our purposes here?

**Comment by andy-haynes** [comment]
@lostpebble are you seeing this error locally when you import the `@wallet-selector/account-export` package? My guess is we missed that it was a CJS package without bundling because it happens to work in `near-wallet` 🤔

**Comment by lostpebble** [comment]
Importing the package worked fine, but it seems like the `require("crypto")` import was just being set to to an empty object or something- since it was only throwing an error when we tried to use that `generator.generate()` function, which used the `crypto` module. I was using Vite in this specific case.

What also could have been happening is that Vite was automatically polyfilling the crypto import with the Web Crypto API: https://developer.mozilla.org/en-US/docs/Web/API/Crypto - and that method just doesn't exist there. The usage is to pass a buffer array of a an already set length to `getRandomValues()`, to be filled with random values.

I also could have polyfilled the Node.js usage using the Vite config- but that's probably not the best dev UX going forward.

I see a fix has already been merged into `dev` 🎉  Will check it out soon.
